src/app/app.module.ts

```diff
@@ -9,7 +9,6 @@ import { CovalentCoreModule } from '@covalent/core';
 import { CovalentHttpModule } from '@covalent/http';
```

Comment:
I saw an error in the console regarding the need to include either BrowserAnimationsModule or NoopAminationsModule. Wondering if you want to include that in the app.modules.